src/main/java/io/percy/appium/metadata/Metadata.java

```diff
@@ -74,6 +82,15 @@ public Integer getStatusBar() {
         return statusBar;
     }
 
+    public Map getSession() {
+        if (Cache.CACHE_MAP.get("getSession_" + sessionId) == null) {
+            driver.addCommand(HttpMethod.GET, "/session/" + driver.getSessionId(), "getSession");
```

Comment:
Tested it, this works for all appium version. no need to use driver.getSessionDetails.